tests/nvidiagpu/deploy-gpu-test.go

```diff
@@ -95,6 +96,15 @@ const (
 	gpuOperatorDefaultMasterBundleImage = "registry.gitlab.com/nvidia/kubernetes/gpu-operator/staging/gpu-operator-bundle:master-latest"
 )
 
+func extractVersionFromCSV(csv string) string {
```

Comment:
May be move this function to the `internal/get/get.go` file and rename it to versionFromCSV() so it can be leveraged by other future test cases.

Reply:
Fixed